File: beecrowd/Java/1008.java
import java.io.IOException;
import java.util.Scanner;

public class Main{
    public static void main(String [] args) throws IOException{
        Scanner keyboard = new Scanner (System.in);

        int funcionario = keyboard.nextInt();
        int horas = keyboard.nextInt();
        double salarioHora = keyboard.nextDouble();

        double salario = horas * salarioHora;

        System.out.printf("NUMBER = %d\nSALARY = U$ %.2f\n", funcionario, salario);

        keyboard.close();
    }
}

File: beecrowd/C++/1008.cpp
#include <iostream>
#include <iomanip:
using namespace std;

int main()
{

    int funcionario;
    cin >> funcionario;

    int horas;
    cin >> horas;

    double salarioHora;
    cin >> salarioHora;

    double salario = horas * salarioHora;

    cout << fixed << setprecision(2);
    cout << "NUMBER = " << funcionario << endl;
    cout << "SALARY = U$ " << salario << endl;
    return 0;
} // end main
